packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -707,8 +730,6 @@ describeIf(process.platform !== 'win32' && !isMacOrWindowsCI)('MongoDB', () => {
       Run prisma generate to generate Prisma Client.
       Prisma schema loaded from prisma/no-model.prisma
       Datasource "my_db"
-
-      Introspecting based on datasource defined in prisma/no-model.prisma …
```

Comment:
Huh what causes this change? Why is the `Prisma schema loaded from prisma/no-model.prisma` and `Datasource "my_db"` before still present?

Reply:
Shouldn't everything from line 717 not be present?

Reply:
So the error throws before we print the log https://github.com/prisma/prisma/pull/11880/files#diff-9eb32f67be2f4d93af4eb94473521226af5a1172b168031ef4002b934e15f21bR199